### PR TITLE
Add a relevant tag into the sample card

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

### What does this PR do?
Add a relevant tag into the sample card.

### What issues does this PR fix or reference?
fixes https://github.com/eclipse/che/issues/19633

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]
Great Job, the only thing that can be improved is displaying the badge starting from the uppercase to make it OperatorHub-like:
image

File: packages/dashboard-frontend/src/pages/GetStarted/GetStartedTab/SamplesListGallery.tsx [hunk 1]
```diff
@@ -50,6 +50,16 @@ type State = {
 };
 
 export class SamplesListGallery extends React.PureComponent<Props, State> {
+  private static compareMetaData(a: che.DevfileMetaData, b: che.DevfileMetaData): number {
```
Comment (akurinnoy) [inline packages/dashboard-frontend/src/pages/GetStarted/GetStartedTab/SamplesListGallery.tsx:53]:
```suggestion
  private static sortByDisplayName(a: che.DevfileMetaData, b: che.DevfileMetaData): number {
```

**Review by akurinnoy — APPROVED** [review]

**Comment by akurinnoy** [comment]
@olexii4 you need to fix this error to have checks passed
```
@eclipse-che/dashboard-frontend: /home/runner/work/che-dashboard/che-dashboard/packages/dashboard-frontend/src/pages/GetStarted/GetStartedTab/SampleCard.tsx
@eclipse-che/dashboard-frontend:   74:53  error  Replace `⏎············{tags}⏎··········` with `{tags}`  prettier/prettier
```

File: packages/dashboard-frontend/src/pages/GetStarted/GetStartedTab/SamplesListGallery.tsx (hunk 1 above)
Comment (olexii4) [reply packages/dashboard-frontend/src/pages/GetStarted/GetStartedTab/SamplesListGallery.tsx]: Thanks. I fixed it.

**Pull request merged**